server/config/util.go

```diff
@@ -66,16 +67,16 @@ func ValidateURLWithScheme(rawURL string) error {
 	return nil
 }
 
-var schedulerMap = make(map[string]struct{})
+var schedulerMap sync.Map
```

Comment:
Why do we need to change it since all register function is called in the `init` function?

Reply:
Because parallelized tests will call [`NewTestOptions`](https://github.com/tikv/pd/blob/e19dc71ac50522bd91ac72052f7c8b39bbb06b3f/server/config/util.go#L83) concurrently which will cause data race on this global `schedulerMap`.